packages/edgeidentity/android/src/main/java/com/adobe/marketing/mobile/reactnative/edgeidentity/RCTAEPEdgeIdentityModule.java

```diff
@@ -39,4 +39,14 @@ public String getName() {
   public void extensionVersion(final Promise promise) {
     promise.resolve(Identity.extensionVersion());
   }
+
+  @ReactMethod
+  public void getExperienceCloudId(final Promise promise) {
+      Identity.getExperienceCloudId(new AdobeCallback<String>() {
```

Comment:
I noticed none of the APIs use AdobeCallbackWithError. If we this instead of AdobeCallback we can also reject the promise with an error in case ECID was not retrieved successfully.
Any reason why we don't do that? cc @yangyansong-adbe

Reply:
https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#getexperiencecloudid
It looks like we also need to update the public docs.

Reply:
@emdobrin For existing wrappers, we only ported the ACP wrapper code to this repository, so we didn't notice that we can use `AdobeCallbackWithError ` in Android code. I have created an issue for it (https://github.com/adobe/aepsdk-react-native/issues/52). I will address it a bit later.

Reply:
Sounds good, let's revisit this for all extensions, for both iOS & Android implementations.